CRM/Civioffice/DocumentRenderer.php

```diff
@@ -154,4 +154,9 @@ public function setConfigItem($name, $value) {
         $this->type->checkConfigurationSupported($name);
         $this->configuration[$name] = $value;
     }
+
+  public function setName(string $name): void {
```

Comment:
What about adding the setter to the base class that actually defines the `name` property?

Reply:
The method is now in the base class.